src/core/coap/coap.cpp

```diff
@@ -239,7 +312,11 @@ otError CoapBase::SendMessage(Message &               aMessage,
                               ResponseHandler         aHandler,
                               void *                  aContext)
 {
+#if OPENTHREAD_CONFIG_COAP_BLOCKWISE_TRANSFER_ENABLE
+    return SendMessage(aMessage, aMessageInfo, TxParameters::GetDefault(), aHandler, aContext, NULL, NULL);
+#else  // OPENTHREAD_CONFIG_COAP_BLOCKWISE_TRANSFER_ENABLE
```

Comment:
nit: can avoid the comments on small blocks (less than 20 lines)
```suggestion
#else
```